app/src/org/commcare/recovery/measures/ScanCczTask.java

```diff
@@ -94,7 +97,19 @@ private void locateAppCcz(File[] files, int depth) {
         String lastKnownCczLocation = HiddenPreferences.getLastKnownCczLocation();
         ArrayList<File> filePathsToScan = new ArrayList<>(3);
         if (!StringUtils.isEmpty(lastKnownCczLocation)) {
-            filePathsToScan.add(new File(lastKnownCczLocation));
+            try {
+                if (lastKnownCczLocation.startsWith("content://")) {
```

Comment:
It's a bad idea to copy the ccz given some of them can be really huge in size. I think we should make the caller of this function to work on file uri instead.